python/utils.py

```diff
@@ -1429,12 +1431,16 @@ def re_sort_per_gene_support(line):
             line[region + '_per_gene_support'] = collections.OrderedDict(sorted(line[region + '_per_gene_support'].items(), key=operator.itemgetter(1), reverse=True))
 
 # ----------------------------------------------------------------------------------------
-def add_linearham_info(sw_info, annotation_list, debug=False):
+def add_linearham_info(sw_info, annotation_list, min_cluster_size=None, debug=False):
     n_already_there = 0
     for line in annotation_list:
+        if min_cluster_size is not None and len(line['unique_ids']) < min_cluster_size:
+            print '       %s adding null linearham info to line: %s, because cluster is less than the passed <min_cluster_size> value of %d' % (color('yellow', 'warning'), ':'.join(line['unique_ids']), min_cluster_size)
+            line['linearham-info'] = null_linearham_info
```

Comment:
I think it would be safer to either use a copy.deepcopy() here, or have a function return it (a la [this](https://github.com/psathyrella/partis/blob/dev/python/indelutils.py#L11)). The way it is now, if this line's linearham info gets subsequently modified, i think it will also modify the linearham info of all other lines that have null info.

Reply:
Ah, you're right. Good catch, thanks!